.ahoy.yml

```diff
@@ -256,25 +256,12 @@ commands:
       ./scripts/drevops/update-drevops.sh $@
   change-uikit:
     usage: Change the version of CivicTheme UI Kit the repository is using
+    aliases: ["bump-uikit"]
     cmd: |
-      pushd tools/civictheme-uikit-version-change
+      pushd scripts/civictheme-uikit-version-manager
       ([ -d "node_modules" ] || npm install)
       npm run start
       popd
-  bump-uikit:
```

Comment:
Removed the first version of the version manager and made this ahoy cmd an alias.